### ISSUE TITLE
VAOS alert message - VVS is unavailable 

### ISSUE THREAD

**Issue by ldelacosta** [department-of-veterans-affairs/va.gov-team#53372]
## Description
When the VVS is unavailable, the system does not return telehealth appointments to the VAOS app which may cause the Veteran to be confused that they are missing their VA appointments. When the frontend receives a response error from the backend services when fetching in person appointments, VAOS should display an alert message to the Veteran. 

---

## Acceptance Criteria
Background: the `va_online_scheduling_vaos_v2_next` toggle is on

Given the user is in the VAOS homepage
When the frontend code receives an error when fetching in person appointments
Then user will see an alert message 
And page content matches the [copy doc]() 

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] Documentation is updated

---

**Comment by ldelacosta** [comment]
@outerpress - Please let us know what the message should state if VVS is unavailable. If you have any questions, please let me know.

**Comment by outerpress** [comment]
We should only show 1 warning or alert in any of these situations - never multiple alerts stacked.

When one appointment type is missing:

```
We can't display [in-person VA|VA video|community care] appointments
We're working to resolve this issue. To manage those appointments, contact the facility where they are scheduled.

[Facility locator](https://staging.va.gov/find-locations)
```
When two appointment types are missing:

```
We can't display [in-person VA|VA video|community care] or [in-person VA|VA video|community care] appointments
We're working to resolve this issue. To manage those appointments, contact the facility where they are scheduled.

[Facility locator](https://staging.va.gov/find-locations)
```

When three appointment types are missing:
I'm assuming there would be no appointments left to display so we'd show the "We're having trouble loading your appointments message. Is that right @ldelacosta ? Or would phone appointments/others still show?

**Comment by ryanshaw** [comment]
@ldelacosta @simiadebowale 

Here's a sample error message that Thomas Joyner supplied:

```json
"failures": [
            {
                "system": "VVS",
                "status": "500",
                "code": 4000,
                "message": "Could not fetch appointments from Video Visit Service",
                "detail": "icn=4420052238V022086, startDate=07/26/2013, endDate=10/27/2013"
            }
        ]
```

**Comment by ldelacosta** [comment]
We implemented this work in #55068, #55280 and #55279. Closing out this ticket as we no longer need it. cc: @JRRoof @outerpress
